css/style.css

```diff
@@ -1039,22 +1041,1372 @@ li { list-style-type: none; }
+    border: 2px solid #fff;
+    color: #fff;
+    background-color: #f00;
+  }
+
+  .info__button:hover {
+    font-size: 17px;
+    color: #fff;
+    border: 3px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+  }
+
+  .info__button:focus {
+    color: #fff;
+    border: 1px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+    outline: none;
+    box-shadow: 0 0 0 3px lightskyblue;
+  }
+
+  .info__button:active {
+    background-color: #fff;
+    border-color: #000;
+    color: #000;
+    font-size: 15px;
+  }
+
+  /* -------3-------- */
+
+  .frame {
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: flex-start;
+    align-content: center;
+    max-width: 768px;
+    height: 1000.25px;
+    margin: 0 auto;
+  }
+    
+  .frame__sidebar {
+    display: flex;
+    order: 2;
+    flex-flow: column wrap;
+    justify-content: flex-start;
+    align-content: flex-start;
+    max-width: 768px;
+    max-height: 487.21px;
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    color: #fff;
+    background: #f00;
+    background-size: 100%;
+  }
+  
+  .frame__title {
+    display: flex;
+    width: 316px;
+    height: 74px;
+    margin: 62.82px 71.88px 0 34px;
+  }
+  
+  .frame__item {
+    display: flex;
+    width: 316px;
+    height: 134px;
+    margin: 12px 71.88px 85.33px 34px;
+  }
+  
+  /* Оценка защиты */
+  
+  .frame__title-rating {
+    display: flex;
+    width: 263px;
+    height: 37px;
+    margin: 62.82px 0 29px 0;
+  }
+  
+  .frame__result {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    width: 295px;
+    height: 173px;
+    font-size: 17px;
+    font-weight: bold;
+    line-height: 21px;
+    list-style: none;
+    margin: 0 0 69.33px 0;
+  }
+  
+  .result-left {
+    display: flex;
+    width: 151px;
+    height: 173px;
+    margin: 0;
+  }
+  
+  li { list-style-type: none; }
+  
+  .result-left__list {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+
+  .result-right {
+    display: flex;
+    width: 140px;
+    height: 173px;
+    padding: 0;
+    margin: 0;
+  }
+  
+  .result-right__list {
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+  
+  .result-right__item {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: center;
+    justify-content: space-between;
+    width: 138px;
+    height: 25px;
+    padding: 0;
+    margin: 0;
+  }
+
+  /* Результат оценки защиты */
+
+  .frame__five-star {
+    display: none;
+  }
+
+  /* ссылка тест и кнопка заказать */
+
+ .frame__info {
+    z-index: 2;
+    position: inherit;
+    width: 459px;
+    height: 55.21px;
+    margin: 0 0 0 -270px;
+  }
+
+    /* правая часть секции, каркас */
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)